draft-demarco-nonce-endpoint.md

```diff
@@ -68,7 +68,9 @@ The **Nonce Endpoint** MUST satisfy the following requirements:
 
 The **Nonce** MUST satisfy the following requirements:
 
-- Pure opacity to recieving Clients.
+- It MUST be opaque to receiving Clients.
+- It MUST be unguessable, by cryptographic randomness or encryption.
```

Comment:
```suggestion
- It MUST be made unguessable through the use of cryptographic randomness.
```

Reply:
I'll take this, but I still think we could be clearer, this is very open ended... which is undesirable IMO.